pkg/v1/cluster/testing/requests_test.go

```diff
@@ -1 +1,166 @@
 package testing
+
+import (
+	"context"
+	"net/http"
+	"reflect"
+	"testing"
+
+	"github.com/selectel/mks-go/pkg/testutils"
+	v1 "github.com/selectel/mks-go/pkg/v1"
+	"github.com/selectel/mks-go/pkg/v1/cluster"
+)
+
+func TestGetCluster(t *testing.T) {
+	endpointCalled := false
+	testEnv := testutils.SetupTestEnv()
+	defer testEnv.TearDownTestEnv()
+
+	testutils.HandleReqWithoutBody(t, &testutils.HandleReqOpts{
+		Mux:         testEnv.Mux,
+		URL:         "/v1/clusters/dbe7559b-55d8-4f65-9230-6a22b985ff73",
+		RawResponse: testGetClusterResponseRaw,
+		Method:      http.MethodGet,
+		Status:      http.StatusOK,
+		CallFlag:    &endpointCalled,
+	})
+
+	ctx := context.Background()
+	testClient := &v1.ServiceClient{
+		HTTPClient: &http.Client{},
+		TokenID:    testutils.TokenID,
+		Endpoint:   testEnv.Server.URL + "/v1",
+		UserAgent:  testutils.UserAgent,
+	}
+	id := "dbe7559b-55d8-4f65-9230-6a22b985ff73"
+
+	actual, httpResponse, err := cluster.Get(ctx, testClient, id)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	expected := testGetClusterResponse
```

Comment:
Looks like `expected` variable is redundant here. Since we have `testGetClusterResponse` constant available in testing package why don't we just use it?
Maybe it can be renamed as `expectedGetClusterResponse` to be more clear